# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/filter-effects/backdrop-filter-basic-background-color.html
# Commit: c20cf2fa829a
# Full Hash: c20cf2fa829a19cb6b60909d05abcda40e77dd70
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-04-20 09:42:47
# Regressor Bug: 1764005
# File Overlap Count: 1
# Description:
#   Bug 1764005 - Add backdrop-filter primitive support r=gfx-reviewers,lsalzman,nical
#   
#   Add support for backdrop-filter primitive in WR. Each backdrop
#   filter primitive establishes a render task sub-graph. The primitive
#   instance registers itself as a resolve source to receive the
# ==============================================================================

diff -r d5b01209cf78 -r c20cf2fa829a testing/web-platform/tests/css/filter-effects/backdrop-filter-basic-background-color.html
--- a/testing/web-platform/tests/css/filter-effects/backdrop-filter-basic-background-color.html	Wed Apr 20 01:08:52 2022 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,34 +0,0 @@
-<!DOCTYPE html>
-<meta charset="utf-8">
-<title>backdrop-filter: Basic operation of filter with background color</title>
-<link rel="author" href="mailto:masonf@chromium.org">
-<link rel="help" href="https://drafts.fxtf.org/filter-effects-2/#BackdropFilterProperty">
-<link rel="match"  href="backdrop-filter-basic-background-color-ref.html">
-
-<div>
-  <p>Expected: A green box and an overlapping blue box, with a purple<br>
-  overlapping region.</p>
-</div>
-<div class="colorbox">
-  <div class="filterbox"></div>
-</div>
-
-<style>
-.colorbox {
-    position: absolute;
-    background: green;
-    width: 100px;
-    height: 100px;
-    left: 10px;
-    top: 100px;
-}
-.filterbox {
-    position: absolute;
-    width: 100px;
-    height: 100px;
-    left: 50px;
-    top: 50px;
-    backdrop-filter: invert(1);
-    background: #00f8;
-}
-</style>
